apps/web/src/components/templates/notification-setting-form/NotificationSettingsForm.tsx

```diff
@@ -137,7 +141,8 @@ export const NotificationSettingsForm = ({
             />
           )}
           <Controller
-            name="notificationGroup"
+            name="notificationGroupId"
+            defaultValue=""
```

Comment:
Could we have set inside of `Controller` `defaultValue = ''`? To avoid all these changes.

Reply:
the `Controller` component is imported from the `react-hook-form`... I don't see the benefit of wrapping it with the defaults if that is the idea, we would need to remember to use that one and remember that by default it has the empty string for that `defaultValue` field...

Reply:
Ok, so if it is the API of an external component we are constricted to it.